AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Request/AuthFederateToIdentityPoolRequest.swift

```diff
@@ -31,19 +31,10 @@ public extension AuthFederateToIdentityPoolRequest {
 
     struct Options {
 
-        
         public let developerProvidedIdentityID: String?
 
-        /// Extra plugin specific options, only used in special circumstances when the existing options do not provide
-        /// a way to utilize the underlying auth plugin functionality. See plugin documentation for expected
-        /// key/values
-        public let pluginOptions: Any?
-
-        public init(
-            developerProvidedIdentityID: String? = nil,
-            pluginOptions: Any? = nil) {
+        public init(developerProvidedIdentityID: String? = nil) {
                 self.developerProvidedIdentityID = developerProvidedIdentityID
```

Comment:
Use of `developerProvidedIdentityID` is not implemented right?

Reply:
Still needs to be implemented.. Coming in the next PR.